akka-actor/src/main/scala/akka/actor/ActorPath.scala

```diff
@@ -155,7 +167,9 @@ object ActorPath {
  */
 @silent("@SerialVersionUID has no effect on traits")
 @silent("deprecated")
-@SerialVersionUID(1L)
+// TODO DOTTY
+// @SerialVersionUID does nothing on a trait
+// @SerialVersionUID(1L)
```

Comment:
I'm not sure why we add an annotation and the silencer to prevent the compiler to say that it's not going to work, any help @raboof @johanandren ?

Reply:
Both the concrete subclasses have their own @SerialVersionUID, I think it is completely fine to remove here as it doesn't do anything anyway?